File: MPI/mpicheck/tests/functions/3.0/MPI_Win_flush_local_all.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Win var_0;
    int ret;
    /* calls */
    ret = MPI_Win_flush_local_all(var_0);
    ret = PMPI_Win_flush_local_all(var_0);
    return 0;
}

File: MPI/mpicheck/tests/functions/3.0/MPI_Win_flush_local_all.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
        call mpi_win_flush_local_all(var_0, var_1)
       call pmpi_win_flush_local_all(var_0, var_1)
        end program main
    